app/jsx/layouts/SeriesLayout.jsx

```diff
@@ -13,14 +13,43 @@ import Form from 'react-router-form'
 // TODO: If UnitSearchLayout is going to be resuscitated, should this layout be wrapped into it?
 
 class SeriesSelector extends React.Component {
+  static propTypes = {
+    unit: PropTypes.shape({
+      id: PropTypes.string.isRequired,
+      name: PropTypes.string.isRequired,
+      type: PropTypes.string,
+      extent: PropTypes.object
+    }).isRequired,
+    series: PropTypes.array.isRequired
+    // unit_id: PropTypes.string.isRequired,
+    // name: PropTypes.string.isRequired
+  }
+ 
+  state = { isOpen: false }
+
+  closeSelector() {
+    this.setState({isOpen: false})
+  }
+
   render() {
+    let p = this.props
     return (
-      <div className="o-input__droplist1">
-        <label htmlFor="c-sort1">Select</label>
-        <select name="" id="c-sort1" value={this.props.unit.id} onChange={(e)=>{browserHistory.push("/uc/"+e.target.value)}}>
-        { this.props.data.series.map((s) => 
-          <option key={s.unit_id} value={s.unit_id}>{s.name}</option>)}
-        </select>
+      <div className="c-campusselector">
```

Comment:
Justin has requested that this selector look just like CampusSelectorComp, but I didn't want to take the time to modify (and rename) CampusSelectorComp to be able to work with a slightly different list. So there's some duplicated code here. Would be nice to refactor someday.